alfresco-transform-tika/alfresco-transform-tika-boot/src/test/resources/quick.3g2_metadata.json

```diff
@@ -2,7 +2,8 @@
   "{http://www.alfresco.org/model/content/1.0}description" : null,
   "{http://www.alfresco.org/model/audio/1.0}releaseDate" : null,
   "{http://www.alfresco.org/model/content/1.0}created" : null,
-  "{http://www.alfresco.org/model/audio/1.0}sampleRate" : "1000",
+  "{http://www.alfresco.org/model/audio/1.0}sampleRate" : "8000",
```

Comment:
I see the sampleRate has changed in 5 'quick' files. What was the need to do that?

Reply:
As I wrote in the PR description, Tika 2.x is more accurate when it comes to data extraction from media containers. The previously used sampling rates were wrong. New values are the same as reported by [ffprobe](https://ffmpeg.org/ffprobe.html)